backend/addcorpus/models.py

```diff
@@ -73,6 +73,12 @@ class CorpusConfiguration(models.Model):
         validators=[validate_markdown_filename_extension],
         help_text='filename of the markdown documentation file for this corpus',
     )
+    citation_page = models.CharField(
+        max_length=128,
+        blank=True,
+        validators=[validate_markdown_filename_extension],
+        help_text='filename of the citation specification (in markdown) for this corpus',
+    )
```

Comment:
Anticipating #1410 , would it not be more suitable to do this through a `FileField`? So that the citation info can be uploaded?

Reply:
I think it makes more sense to keep using the same system that we use for descriptions, images and word model documentation. If we want to adjust it later, we need to adjust those three fields anyway.